src/View/Admin/index.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+    <h1 class = "display-1" >Bienvenue Admin_name !</h1>
+    <p><a href="admin/add">Ajouter une musique</a></p>
+
+<table class = "table">
+    <thead>
+        <tr>
+            <th>Titre</th>
+            <th>Auteur</th>
+            <th>Genre</th>
+            <th>Source</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for music in musics %}
+    {% set musical_genres = ['pop','cinematic','electro','world','chillout','ambient'] %}
```

Comment:
No this list should not be hardcoded, but come form your database and shared thanks to your controller